### PR TITLE
adds pom to gitignore

### PR DESCRIPTION
**Pull request by mau-fyi**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,8 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.google.sps</groupId>
-  <artifactId>summer22-sps-36</artifactId>
+  <!-- summer22-sps-36, mauricio-sps-summer-22-team-36 -->
+  <artifactId>mauricio-sps-summer-22-team-36</artifactId>
```
Comment (sbahaud) [inline pom.xml]: Maybe comment this out and only submit the main one.

File: pom.xml [hunk 2]
```diff
@@ -16,7 +17,8 @@
 
     <!-- Project-specific properties -->
     <exec.mainClass>com.google.sps.app.ServerMain</exec.mainClass>
-    <googleCloudProjectId>summer22-sps-36</googleCloudProjectId>
+    <!-- summer22-sps-36, mauricio-sps-summer-22-team-36 -->
```
Comment (sbahaud) [inline pom.xml]: Same here

**Review by sbahaud — APPROVED** [review]
Approved with comments

File: pom.xml (hunk 1 above)
Comment (mau-fyi) [reply pom.xml]: Done

File: pom.xml (hunk 2 above)
Comment (mau-fyi) [reply pom.xml]: done

**Pull request merged**